global.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "sdk": {
-    "version": "3.0.100"
+    "version": "3.0.100",
```

Comment:
Is it worth going to 3.1 (LTS) in the same PR? I don't know how difficult that is.

Reply:
Agreed that we should upgrade the dotnet SDK, but I'd rather do that in a separate PR.